### ISSUE TITLE
Dancing Scimitar is prompting to reveal Bear Strength

### ISSUE THREAD

**Issue by sIKE23** [sIKE23/Mage-Wars#284]
I thought that we had this fixed!

**Comment by ACG8** [comment]
Well, we have the Scimitar Buff fixed; note that revealing Bear Strength doesn't actually buff the Scimitar's attack. I agree that the prompt is misleading, though. I'll see about making a temporary patch, and then will find a better way to handle enchantment prompts in the long term.

**Comment by ACG8** [comment]
Okay, on reflection, fixing this even just for the scimitar is not trivial, and will require a substantial rewrite to the enchantment prompt system. Currently, the system just looks at which enchantments would be appropriate to reveal at the current battle step, without taking into consideration any of the context. I will have to rewrite it to take more things into account. How critical is this fix?

**Comment by sIKE23** [comment]
It is marked as a Q3 fix, if you want to update the topic so it addresses a
larger fix then I am fine with that. You wanted to re-write this anyways
was my understanding.

//FC

**Comment by ACG8** [comment]
Yes, that's right. Another thing I want to add is the prompts for enchantments that trigger under conditions other than combat (such as nullify).
